cmd/generate.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (C) 2021-2025, Kubefirst
+
+This program is licensed under MIT.
+See the LICENSE file for more details.
+*/
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/konstructio/kubefirst/internal/generate"
+	"github.com/konstructio/kubefirst/internal/progress"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func GenerateCommand() *cobra.Command {
+	generateCommand := &cobra.Command{
+		Use:   "generate",
+		Short: "code generator helpers",
+	}
+
+	// wire up new commands
+	generateCommand.AddCommand(generateApp())
+
+	return generateCommand
+}
+
+func generateApp() *cobra.Command {
+	var name string
+	var environments []string
+	var outputPath string
+
+	appScaffoldCmd := &cobra.Command{
+		Use:              "app-scaffold",
+		Short:            "scaffold the gitops application repo",
+		TraverseChildren: true,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			if err := generate.AppScaffold(name, environments, outputPath); err != nil {
+				progress.Error(err.Error())
+				return errors.Wrap(err, "error scaffolding app")
+			}
+
+			progress.Success(fmt.Sprintf("App successfully scaffolded: %s", name))
+			return nil
+		},
+	}
+
+	cwd, err := os.Getwd()
+	cobra.CheckErr(err)
```

Comment:
You don't need to get the current working directory if the path to the flag starts with `./`.

The entire std library (with a few exceptions) can handle `./` whenever it's file opening and all that.